packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.jsx

```diff
@@ -297,13 +301,15 @@ const FinancialAssessment = ({
 
     const getScrollOffset = () => {
         if (isMobile()) return is_appstore ? '160px' : '200px';
+        if (isMobile()) return is_mf && is_financial_information_incomplete && !is_submit_success ? '40px' : '80px';
+        if (isDesktop()) return is_mf && is_financial_information_incomplete && !is_submit_success ? '140px' : '80px';
```

Comment:
Can you combine two conditions? If isMobile() is false, I think it's expected that isDesktop() is true, unless we have other values... So we can omit calling it